### DISCUSSION TITLE
enviroment variables list?

### DISCUSSION THREAD

**Discussion by migs35323** [keycloak/keycloak#16129] [Q&A]
trying to setup a keycloak in a docker compose i am having trouble setting up a database, what are the available/correct enviroment variables i can use to configure the container?
i am having trouble with the postgres database, it is defaulting to use the localhost address which is not correct...

**Comment by torgerss** [comment]
The documentation for containers is severely lacking in any sort of useful detail.  Great, you documented some basic steps to get a container going.  The rest of the functionality and setting variables is non-existant.  Actually, lets be honest and just say the container documentation for the new 18+ versions of Keycloak suck.  Great detail for everything under the sun when running on a server a fat application, but nothing for configuration of environment variables when running in Docker container with a compose file or even CLI options which would use the same.

**Reply by ksaadDE** [reply]
> lets be honest and just say the container documentation for the new 18+ versions of Keycloak suck. 

Just wanted to post that. But you were first. xD 

Any ideas for docker-compose ??
